packages/browser-wallet/src/wallet-api/compatibility.ts

```diff
@@ -222,7 +224,10 @@ export interface ConfigureDelegationPayloadV0 extends Omit<ConfigureDelegationPa
 export type ConfigureDelegationPayloadCompat = ConfigureDelegationPayloadV0 | ConfigureDelegationPayload;
 
 export type RegisterDataPayloadCompat = RegisterDataPayload;
-export type UpdateCredentialsPayloadCompat = UpdateCredentialsPayload;
+export type UpdateCredentialsPayloadCompat =
+    | UpdateCredentialsPayload
+    | UpdateCredentialsInput
+    | UpdateCredentialKeysInput;
```

Comment:
I feel like this should be just
```suggestion
export type UpdateCredentialsPayloadCompat = UpdateCredentialsInput;
```

?